tools/clang/tools/dxcompiler/dxclinker.cpp

```diff
@@ -415,6 +415,11 @@ HRESULT STDMETHODCALLTYPE DxcLinker::Link(
             SerializeFlags, pOutputStream, 0, opts.DebugFile, &Diag,
             &ShaderHashContent, pReflectionStream, pRootSigStream, nullptr,
             nullptr);
+
+        if (opts.StripDebug) {
+          inputs.pM->GetOrCreateDxilModule().StripNamesSensitiveToDebug();
+        }
```

Comment:
Modern LLVM omits braces for single-line compound statements, and DXC follows that convention:

```suggestion
        if (opts.StripDebug)
          inputs.pM->GetOrCreateDxilModule().StripNamesSensitiveToDebug();
```
See: [LLVM Coding Standards: Don't use braces on single-line statements](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)